browser/src/layer/tile/CanvasTileLayer.js

```diff
@@ -1886,6 +1886,12 @@ L.CanvasTileLayer = L.Layer.extend({
 		else if (textMsg.startsWith('readonlyhyperlinkclicked: ')) {
 			var jsonString = textMsg.replace('readonlyhyperlinkclicked: ', '');
 			var json = JSON.parse(jsonString);
+
+			if (this._docType === 'spreadsheet') {
+				json.position.x *= app.pixelsToTwips;
+				json.position.y *= app.pixelsToTwips;
```

Comment:
core sends position in pixels? it seems to be incorrect, what if user has different DPI in the browser than server's view when generating message?